README.md

```diff
@@ -50,36 +50,24 @@ pluginManagement {
 }
 ```
 
-`build.gradle`
+`build.gradle.kts`
 
-```groovy
+```kotlin
 plugins {
-    id("org.jetbrains.kotlin.jvm") version "1.9.0"
-    id("com.google.devtools.ksp") version "1.9.0-1.0.13"
+    id("org.jetbrains.kotlin.jvm")
+    id("com.google.devtools.ksp")
 }
 
 repositories {
     mavenCentral()
     google()
 }
 
-dependencies {
-    ksp("me.tatarka.inject:kotlin-inject-compiler-ksp:0.6.3")
-    implementation("me.tatarka.inject:kotlin-inject-runtime:0.6.3")
-}
-```
-
-### or with KAPT (deprecated)
-
-```groovy
-plugins {
-  id("org.jetbrains.kotlin.jvm") version "1.9.0"
-  id("org.jetbrains.kotlin.kapt") version "1.9.0"
-}
```

Comment:
KAPT support was removed (although the latest stable still has it)

Reply:
yeah, I plan to do a README refresh with the next release, It's the first thing you see when navigating to the project on github so I don't love updating it before a release is actually out.

Reply:
I reverted the changes to the README